.github/workflows/ci.yml

```diff
@@ -107,11 +107,11 @@ jobs:
         run: ./ci/run_tests.sh build_nvim
 
       - if: "!cancelled()"
-        name: clint-full
-        run: ./ci/run_lint.sh clint-full
+        name: lintcfull
+        run: ./ci/run_lint.sh lintcfull
 
       - if: "!cancelled()"
-        name: stylua
+        name: lintstylua
```

Comment:
why isn't this part of `lintlua` ?

Reply:
Wait, this becomes super difficult without having an external script, because we'd need four separate add_custom_target for all cases if stylua/luacheck exists/doesn't exist.